packages/css/src/components/card/card.scss

```diff
@@ -14,7 +14,7 @@
   &:has(:focus-visible) {
     outline-color: -webkit-focus-ring-color;
     outline-style: auto;
-    outline-width: 1px;
```

Comment:
That `0.0625rem` leads to a 1px width with a default 16px font-size is not immediately obvious, while `1px` is. So, I wonder if we could use functions that calculate the rem value from a given pixel value?

Reply:
Why do we use `rem` and not `em` for margins and paddings?

Reply:
> That 0.0625rem leads to a 1px width with a default 16px font-size is not immediately obvious, while 1px is. So, I wonder if we could use functions that calculate the rem value from a given pixel value?

I don’t think it’s very important to know to how many pixels something comes down to. Also, 1rem is not always equal to 16px. We want to express lengths as a fraction of the user’s font size. I agree that `0.5rem` is clearer than `0.0625rem`, but give it time and you’ll know them all by heart like me :)

Another reason not to: most values live in tokens, and we don’t want Sass function inside of them, and we can’t wrap them in CSS or we must check whether a value is expressed in pixels or rems – or cqws etc.

Reply:
> Why do we use rem and not em for margins and padding?

Because margins and paddings are defined on containers and font-size is not, so ems and rems are often the same there, and ems are less predictable.